packages/components/src/foldable-card/index.jsx

```diff
@@ -48,6 +49,7 @@ class FoldableCard extends Component {
 		expanded: false,
 		screenReaderText: false,
 		smooth: false,
+		useInert: true,
```

Comment:
For backwards compatibility, shouldn't the default value be `false`, that way we don't alter the behavior of the existing Foldable Cards? Though I suppose if a card is collapsed, we likely wouldn't need keyboard navigation for it 🤔

In any case, right now the effective behavior seems to me identical as in your [other draft PR](https://github.com/Automattic/wp-calypso/pull/102061/files), though I could be missing something.

If this is intentional, we could look at all the other existing usages of Foldable Card and make sure that everything is working as expected.

Reply:
Ah, I meant to default this to `false`. 😅 Good catch! Updated in 8641df155ab3e9708c967cb69f61fa3494579056

I'm open to the "breaking change" [approach](https://github.com/Automattic/wp-calypso/pull/102061/files) (provided we test with some confidence that it won't actually break anything). I propose we start with this backward compatible approach since it's safe and will unblock the Reader Editor fastest. WDYT?

Reply:
Sounds good to me!